styles/styles.css

```diff
@@ -0,0 +1,325 @@
+  height: 40px;
+  background-image: url("../image/header/navigation.png");
+  margin: 0;
+  background-repeat: no-repeat;
+}
+
+.navigation {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 198px;
+}
+
+.navigation--links {
+  display: flex;
+  gap: 48px;
+}
+
+.navigation--links__block {
+  display: flex;
+  gap: 24px;
+  align-items: center;
+}
+
+nav {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 28px;
+  text-align: center;
+}
+
+nav a {
+  text-decoration: none;
+  color: #ffffff;
+}
+
+nav a:nth-child(1) {
+  position: relative;
+  line-height: 44px;
+}
+
+nav a:nth-child(1):after {
+  content: "";
+  position: absolute;
+  width: 50px;
+  height: 4px;
+  background-color: #f5bd41;
+  transform: translateX(10%);
+  margin: 0 auto;
+  display: block;
+}
+
+.navigation--mode {
+  width: 30px;
+  height: 30px;
+  background-image: url("../image/header/navigation.png");
+  background-position: 620px;
+}
+
+.navigation--login {
+  width: 40px;
+  height: 40px;
+  background-image: url("../image/header/navigation.png");
+  background-position: 590px;
+}
+
+.main-title {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 50px;
+  line-height: 59px;
+  color: #ffffff;
+  margin-bottom: 97px;
+}
+
+.search {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-end;
+  margin-bottom: 120px;
+}
+
+.search p {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  align-content: flex-start;
+}
+
+label {
+  padding-left: 25px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 31px;
+  text-align: left;
+  color: #ffffff;
+}
+
+input {
+  display: flex;
+  height: 64px;
+  background: #ffffff;
+  border: 3px solid #f3f3f4;
+  border-radius: 8px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 21px;
+  color: #383838;
+}
+
+.search--name {
+  display: flex;
+  flex-grow: 13;
+}
+
+.search--date {
+  display: flex;
+  flex-grow: 2;
+}
+
+.search--bar {
+  display: flex;
+  flex-grow: 5;
+}
+
+.search--button {
+  display: flex;
+  flex-grow: 7;
+}
+
+input[type="text"] {
+  padding-left: 24px;
+}
+
+#date {
+  border: 3px solid #f5bd41;
+  border-radius: 8px;
+}
+
+input[type="submit"] {
+  display: flex;
+  justify-content: center;
+  background: #3077c6;
+  border: 0;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 28px;
+  color: #ffffff;
+}
+
+.app {
+  width: 270px;
+  height: 40px;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+
+}
+
+.app-android {
+  width: 50%;
+  height: 100%;
+  background-image: url("../image/header/groupapp.png");
+  background-position: left;
+  background-repeat: no-repeat;
+}
+
+.app-apple {
+  width: 44.5%;
```

Comment:
It is not a good idea to calculate this value in relative units. 
This block with icons could be more flexible regardless of the content. What happens if we add the 3-rd icon to it?